docs/api-usage/slug-fields.mdx

```diff
@@ -11,7 +11,7 @@ Slug can contain:
 - underscores
 - hyphens
 
-Slugs are not translatable in Saleor. When using different languages, users should apply the same product URLs regardless of the language.
+Slugs are translatable in Saleor for product, categories, collections and pages. When using different languages, users can apply the same product URLs regardless of the language or make use translation specific slugs.
```

Comment:
Rewrite suggestion:

Slugs in Saleor are translatable for products, categories, collections, and pages. This allows users to either maintain consistent URLs across different languages or use language-specific slugs for better localization.